exporter/elasticsearchexporter/factory.go

```diff
@@ -79,3 +82,18 @@ func createLogsExporter(
 		exporterhelper.WithShutdown(exporter.Shutdown),
 	)
 }
+
+func createTracesExporter(ctx context.Context,
```

Comment:
Please add a warning message if the deprecated field is being used

Reply:
make sense